ember/app/logout/route.js

```diff
@@ -0,0 +1,15 @@
+import Route from "@ember/routing/route";
+import { inject as service } from "@ember/service";
+import Ember from "ember";
+
+export default class LogoutRoute extends Route {
```

Comment:
Why do you need a route for this if you can just use the action?

Reply:
I removed the route and used an action.